src/Infrastructure/Caching/ApplicationEasyRedisCacheManager.cs

```diff
@@ -0,0 +1,48 @@
+﻿using CleanArchitecture.Application.Common.Interfaces;
+using EasyCache.Redis.Concrete;
+using Microsoft.Extensions.Caching.Distributed;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CleanArchitecture.Infrastructure.Caching
+{
+    public class ApplicationEasyRedisCacheManager : EasyCacheRedisManager, IApplicationCacheService
+    {
+        public ApplicationEasyRedisCacheManager(IDistributedCache distributedCache) : base(distributedCache)
+        {
+        }
+
+        public override T Get<T>(string key)
```

Comment:
Why do you override if you don't change anything?

Reply:
You are right @enisn. Can you check commit 445f43218a3465130622cd39569ad9640aa46077 ?